### ISSUE TITLE
Error when training pre-trained mothalf.pth model

### ISSUE THREAD

**Issue by RanaMostafaAbdElMohsen** [PeizeSun/TransTrack#17]
I tried to load `mothalf.pth` **pre-trained** model in `train mode` with same default parameters used in the `main_track.py`

The program crashes with error below:

Exception has occurred: ValueError
loaded state dict contains a parameter group that doesn't match the size of optimizer's group
  File "/home/rana/Documents/Research_Projects/Transformers/TransTrack/main_track.py", line 265, in main
    optimizer.load_state_dict(checkpoint['optimizer'])
  File "/home/rana/Documents/Research_Projects/Transformers/TransTrack/main_track.py", line 366, in <module>
    main(args)

Are there different parameters used for mothalf.pth model other specified in main_track.py? Please advice.

**Comment by PeizeSun** [comment]
Hi~
If a trained model is used to re-train(instead of resume), it needs to be converted by:
`python track_tools/crowdhuman_model_to_mot.py --pre_model mothalf.pth --saved_model mothalf_ready.pth
`
